src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,30 @@
+.calendar {
+  padding: 15px $padding;
+  width: $count-width;
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+
+  &__day {
+    display: flex;
+
+    background-color: #eee;
+    border: 1px solid #000;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    justify-content: center;
+    align-items: center;
+
+    @include mix-for-box($box-size);
+    transition: transform 0.5s;
```

Comment:
Move it up by `20px` and bg color: both of these properties are to be animated with the duration of half a second